From 58e9242638a46618b020aa3d39282ab692433265 Mon Sep 17 00:00:00 2001
From: Ryan Volz <rvolz@mit.edu>
Date: Fri, 10 Aug 2018 18:07:17 -0400
Subject: [PATCH] Fix build RPATH remaining on installed libpyuhd by not
 setting it.

---
 host/python/CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/host/python/CMakeLists.txt b/host/python/CMakeLists.txt
index d0a6e2204..ed4d92b57 100644
--- a/host/python/CMakeLists.txt
+++ b/host/python/CMakeLists.txt
@@ -29,6 +29,9 @@ target_include_directories(pyuhd PUBLIC
 )
 
 target_link_libraries(pyuhd ${BOOST_PYTHON_LIBRARY} ${Boost_LIBRARIES} ${PYTHON_LIBRARY} uhd)
+# skip setting pyuhd's RPATH because it is installed as a python data file so
+# the build RPATH would never be overwritten by an install RPATH
+set_target_properties(pyuhd PROPERTIES SKIP_BUILD_RPATH ON BUILD_WITH_INSTALL_RPATH OFF)
 # Copy pyuhd library to the staging directory
 if(WIN32)
     set(PYUHD_LIBRARY_NAME libpyuhd.pyd)
-- 
2.20.1

